src/styles/booleanDropdown.scss

```diff
@@ -33,21 +35,49 @@
   background-color: lightgray;
 }
 
-.arrow {
+.arrow-closed {
+  background: url('../assets/arrow.svg') no-repeat right;
+  display: inline-block;
+  padding: 4px;
+}
+
+.arrow-open {
   background: url('../assets/arrow.svg') no-repeat right;
   display: inline-block;
   padding: 4px;
+  transform: scaleY(-1);
 }
 
 .Dropdown-arrow-wrapper {
   padding-right: 7px;
 }
 
 .Dropdown-control {
+  border-radius: 3px;
   display: flex;
+  min-height: 27px;
+  min-width: 60px;
+  padding-left: 10px;
+}
+
+.Dropdown-control:hover {
+  background-color: #adadad;
```

Comment:
this seems to be the general hover color (even in `levelSelect.scss`), can we pull it out and put it in `global.scss`?